contracts/sdk/indexer/db.ts

```diff
@@ -323,14 +484,16 @@ export class NFTDatabaseConnection {
   }
 
   async generateProof(
+    treeId: string,
     nodeIdx: number,
     hash: Buffer,
     dataHash: string,
     creatorHash: string,
     nonce: number,
     owner: string,
     delegate: string,
-    check: boolean = true
+    check: boolean = true,
+    seq: number | null = null
```

Comment:
add a comment:
`/// if seq is given: creates tree made from sequence numbers <= seq`